src/v4l2_camera.cpp

```diff
@@ -88,10 +166,35 @@ V4L2Camera::V4L2Camera(rclcpp::NodeOptions const & options)
   auto timestamp_offset = declare_parameter<int64_t>("timestamp_offset", 0, timestamp_offset_descriptor);
   rclcpp::Duration timestamp_offset_duration = rclcpp::Duration::from_nanoseconds(timestamp_offset);
 
+  // Prepare diagnostics
+  auto hardware_id = declare_parameter<std::string>("hardware_id", "");
+  min_ok_rate_ = declare_parameter<double>("min_ok_rate", 9.0);
+  max_ok_rate_ = declare_parameter<double>("max_ok_rate", 11.0);
+  min_warn_rate_ = declare_parameter<double>("min_warn_rate", 8.0);
+  max_warn_rate_ = declare_parameter<double>("max_warn_rate", 12.0);
+  num_frames_transition_ = declare_parameter<int>("num_frames_transition", 3);
```

Comment:
Should we change this to "observed_frames_transition" to match the diagnostics?

Reply:
I renamed the parameter and the corresponding variable for better consistency in 29c3410. Thank you for pointing this out.